app/src/main/java/com/rentminder/MainActivity.kt

```diff
@@ -128,14 +126,18 @@ fun EditBillAmounts() {
     val keyboardController = LocalSoftwareKeyboardController.current
     val focusManager = LocalFocusManager.current
 
-    //Column for all bill rows
+    /**
+     * Column for all bill rows
+     */
```

Comment:
Proper KDoc format should be used above method names, as shown here.  Inline comments should not be used above a method or class.